deftree/svcparse/scanner_test.go

```diff
@@ -49,3 +49,39 @@ func TestScanSingleLineComments(t *testing.T) {
 
 	}
 }
+
+func TestScanMultiLineComments(t *testing.T) {
+	r := strings.NewReader("service testing\n /* comment1 */\n/*comment2 */\n\n/*comment 3 */\n what")
+	scn := NewSvcScanner(r)
+	for i, good_str := range []string{
+		"service",
+		" ",
+		"testing",
+		"\n ",
+		"/* comment1 */",
+		"\n",
+		"/*comment2 */",
+		"\n\n",
+		"/*comment 3 */",
+		"\n ",
+		"what",
+		"",
+	} {
+		unit, err := scn.ReadUnit()
+		str := string(unit)
+
+		if str != good_str {
+			for _, grp := range scn.Buf {
+				t.Logf("  '%v'\n", cleanStr(string(grp.Value)))
+			}
+			t.Fatalf("%v returned token '%v' differs from expected token '%v'\n", i, cleanStr(str), cleanStr(good_str))
+		}
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			t.Fatalf(fmt.Sprintf("%v", err))
```

Comment:
You ask this as though I didn't just copy-paste most of this test from another test, one which was written months ago at the start of my Go knowledge : )

But yes, I should change those.